lib/dialects/abstract/query-generator.js

```diff
@@ -1111,7 +1111,11 @@ var QueryGenerator = {
           attributes = include.attributes.map(function(attr) {
             var attrAs = attr,
                 verbatim = false;
-
+            
+            if (include.model && include.model._isGeometryAttribute(attr)) {
+                attr = self.geometrySelect(attr,include.as);
```

Comment:
Please add spaces between the comma and the next argument - this also applies to function definitions